dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Sleep/SleepDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { NodeRun, SleepNode } from 'littlehorse-client/proto'
+import { FC } from 'react'
+import { NodeDetails } from '../NodeDetails'
+import { formatDate } from '../../../../../utils'
+import { getVariable, formatTime } from '@/app/utils'
+
+export const SleepDetails: FC<{ sleepNode?: SleepNode; nodeRun?: NodeRun }> = ({ sleepNode, nodeRun }) => {
+  if (!sleepNode) return
+  const timeValue = getVariable(sleepNode?.rawSeconds)
+  return (
+    <NodeDetails>
+      <div className="mb-2">
+        <div className="flex items-center gap-1 whitespace-nowrap text-nowrap">
+          <h3 className="font-bold">SleepNode</h3>
+        </div>
+        <div className="flex flex-col gap-2 text-nowrap">
+          <div className="flex ">
+            {sleepNode.rawSeconds && <div>Time: {typeof timeValue === 'number' ? formatTime(timeValue) : timeValue}</div>}
```

Comment:
so as per ts schema . it can be VariableAssignment, and we are calling formattime function which except a int
![image](https://github.com/user-attachments/assets/78048570-ec45-401c-8057-b1dd3552b8e1)